pkg/jobstore/types.go

```diff
@@ -79,6 +79,12 @@ type Store interface {
 	// GetEvaluation retrieves the specified evaluation
 	GetEvaluation(ctx context.Context, id string) (models.Evaluation, error)
 
+	// GetEvaluationsByState returns all evaluations in a specific state
+	GetEvaluationsByState(ctx context.Context, state string) ([]models.Evaluation, error)
```

Comment:
Nope. There's another PR that enqueues evaluations from a watcher, and I am going to have it return all of the 'pending' evaluations when calling Watch() - so that at startup we'll get all the evaluations in the order they were created as if they had just been created.  It's just that until we have different states for the evaluations, it would return all the evaluations ever each time we start up.